### DISCUSSION TITLE
Showcase and Project Archetypes

### DISCUSSION THREAD

**Discussion by thescientist13** [ProjectEvergreen/www.greenwoodjs.dev#19] [General]
## Overview

It would be nice to have some good examples of project's to help showcase what Greenwood can do, as captured in #4 but it might take more time to build up a robust showcase before the home page is ready, so just opening up the conversation here to keep that rolling in the back of our minds, or a goal to work towards after the main website goes live.

## Archetypes

Here are some that come to mind

1. Blog / Portfolio (SSG)
1. E-Commerce (SSR)
1. Netflix clone (SPA)
1. HackerNews clone (SSR)

> Ideally I would like a Showcase project to actually be a live in production project, but I like the idea of having "clone" repos as a fallback though just to at least help demonstrate that Greenwood _can_ do that.

## Templates / Starters

Greenwood has an init CLI which can be used to scaffold out new projects from a GitHub repo.  Currently the only option it has besides a bare repo is a Blog.  What could other good examples be?
1. SPA
    - but then this would need us to pick a router, probably a lib, etc).  But then we have to maintain it...
    - Maybe **greenwood-spa-lit**?
1. ???

## Ecosystem / Community Outreach 
1. Vercel Supported frameworks list
1. USWDS - https://github.com/uswds/uswds/discussions/5922
1. Are WCs a thing yet - https://arewebcomponentsathingyet.com/

**Comment by Auhseh** [comment]
I think it’ll be a great idea to have High quality production ready scaffold projects 

There seems to be existing market demand from Backend devs looking to build their frontends and Very early stage startups looking to move fast.

Initially, we can start with:
- A simple Saas SPA:

* @lit-labs/router for routing
    * Basic layout (header, footer, main content)
    * Example pages; landing, pricing, features etc.)
    * Login/Sign up
    * App page/ user settings/profiles
    * Toast/modal/loading
    * Dashboard/analytics (if needed)

- A small business/landing boilerplate
- A simple Portfolio boilerplate

**Comment by thescientist13** [comment]
Something to keep in mind as part of integrating with the `@greenwood/init` were GitHub rate limiting issues we saw with our previous implementation, which we should try and account for somehow - https://github.com/ProjectEvergreen/greenwood/issues/1078
